WISE-Community/WISE-Client
# feat(Multilingual Unit): Translate HTML component

## Changes
- Create TranslatableRichTextEditorComponent for translating HTML components in the Authoring Tool
   - if in default language mode, show one rich text editor
   - if in another language, show translation text and default language text in rich text editors in their own tabs

## Notes
- To limit the scope of this review, these will be implemented in separate PR's:
   - Disable editing in the rich text editor in the default language tab 
   - Add "Copy to [Japanese,Spanish,...]" button in the default language tab that will copy text into the translation language tab

## Test prep
- Test with https://github.com/WISE-Community/WISE-API/pull/253 

## Test (in the AT)
- In default language mode, editing HTML component should work as before
- In another language mode, you can
   - see and edit the language's HTML content in the first tab
   - see default language's HTML content in the second tab
- Try switching between the various languages and make sure the contents appear and save correctly
- Try starting with an empty HTML component, to test out cases where translations don't exist in translation files
